[2023-06-10T02:01:14.331+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2020-12-02T06:30:00+00:00 [queued]>
[2023-06-10T02:01:14.344+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2020-12-02T06:30:00+00:00 [queued]>
[2023-06-10T02:01:14.344+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T02:01:14.345+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-10T02:01:14.346+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T02:01:14.364+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2020-12-02 06:30:00+00:00
[2023-06-10T02:01:14.372+0000] {standard_task_runner.py:54} INFO - Started process 3627 to run task
[2023-06-10T02:01:14.379+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data', 'format_to_parquet_task', 'scheduled__2020-12-02T06:30:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpbvbxd2rv']
[2023-06-10T02:01:14.383+0000] {standard_task_runner.py:83} INFO - Job 176: Subtask format_to_parquet_task
[2023-06-10T02:01:14.389+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-10T02:01:14.473+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-10T02:01:14.496+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-10T02:01:14.549+0000] {task_command.py:384} INFO - Running <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2020-12-02T06:30:00+00:00 [running]> on host f41d61956f54
[2023-06-10T02:01:14.587+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-10T02:01:14.623+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=green_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-02T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-02T06:30:00+00:00
[2023-06-10T02:01:14.751+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_multiFile_v02.py", line 33, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 886, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 895, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 114, in pyarrow.lib.check_status
OSError: zlib inflate failed: incorrect header check
[2023-06-10T02:01:14.766+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=green_taxi_data, task_id=format_to_parquet_task, execution_date=20201202T063000, start_date=20230610T020114, end_date=20230610T020114
[2023-06-10T02:01:14.781+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 176 for task format_to_parquet_task (zlib inflate failed: incorrect header check; 3627)
[2023-06-10T02:01:14.841+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-10T02:01:14.864+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-06-11T01:14:43.741+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2020-12-02T06:30:00+00:00 [queued]>
[2023-06-11T01:14:43.749+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2020-12-02T06:30:00+00:00 [queued]>
[2023-06-11T01:14:43.750+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:14:43.750+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-11T01:14:43.750+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:14:43.775+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2020-12-02 06:30:00+00:00
[2023-06-11T01:14:43.781+0000] {standard_task_runner.py:54} INFO - Started process 4709 to run task
[2023-06-11T01:14:43.789+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data', 'format_to_parquet_task', 'scheduled__2020-12-02T06:30:00+00:00', '--job-id', '503', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmp9jw7yp4x']
[2023-06-11T01:14:43.792+0000] {standard_task_runner.py:83} INFO - Job 503: Subtask format_to_parquet_task
[2023-06-11T01:14:43.794+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-11T01:14:43.946+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-11T01:14:43.992+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-11T01:14:44.088+0000] {task_command.py:384} INFO - Running <TaskInstance: green_taxi_data.format_to_parquet_task scheduled__2020-12-02T06:30:00+00:00 [running]> on host f41d61956f54
[2023-06-11T01:14:44.137+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-11T01:14:44.192+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=green_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-02T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-02T06:30:00+00:00
[2023-06-11T01:14:44.532+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-11T01:14:44.555+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=green_taxi_data, task_id=format_to_parquet_task, execution_date=20201202T063000, start_date=20230611T011443, end_date=20230611T011444
[2023-06-11T01:14:44.622+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-11T01:14:44.650+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
